### PR TITLE
A temporary fix for column width

### PR DESCRIPTION
**Pull request by Akarshit**
I have changed the text from "Version 1.0.2 available" to "v1.0.2 available"and also tried to shorten the version.
This is a temporary fix for #3609

### CODE REVIEW

File: app/src/processing/app/contrib/StatusPanel.java [hunk 1]
```diff
@@ -235,9 +236,9 @@ public void update(ContributionPanel panel) {
       latestVersion = "Update";
     }
 
-    if(currentVersion != null){
-      currentVersion = "Version " + currentVersion;
-    }else{
+    if (currentVersion != null) {
+      currentVersion = "v" + currentVersion;
```
Comment (benfry) [inline app/src/processing/app/contrib/StatusPanel.java]: This makes an assumption about the format of the version field that may not always be correct. Putting "v" in front won't make sense in some cases. Just leave it off.

File: app/src/processing/app/contrib/ContributionListPanel.java [hunk 2]
```diff
@@ -32,6 +32,8 @@
 import javax.swing.event.*;
 import javax.swing.table.*;
 
+import com.sun.xml.internal.bind.v2.runtime.reflect.Lister.Pack;
```
Comment (benfry) [inline app/src/processing/app/contrib/ContributionListPanel.java]: Why is this here?

**Comment by benfry** [comment]
I'll wait until you have the final fix.

File: app/src/processing/app/contrib/StatusPanel.java (hunk 1 above)
Comment (Akarshit) [inline app/src/processing/app/contrib/StatusPanel.java]: @benfry Do you want the label to read "1.0.1 available" only ?

File: app/src/processing/app/contrib/ContributionListPanel.java (hunk 2 above)
Comment (Akarshit) [inline app/src/processing/app/contrib/ContributionListPanel.java]: My bad.

File: app/src/processing/app/contrib/StatusPanel.java (hunk 1 above)
Comment (benfry) [inline app/src/processing/app/contrib/StatusPanel.java]: Yes, but via `Language.interpolate()`, and make the string "%s available" so that it can be localized to languages that might need the equivalent of "available" in front.
